src/plugins/saved_objects_management/public/management_section/objects_table/components/table.tsx

```diff
@@ -402,7 +402,8 @@ export class Table extends PureComponent<TableProps, TableState> {
               color="danger"
               onClick={onDelete}
               isDisabled={
-                selectedSavedObjects.length === 0 || !capabilities.savedObjectsManagement.delete
+                selectedSavedObjects.filter(({ managed }) => !managed).length === 0 ||
```

Comment:
Could consider adding an on hover message on the button for this case. Something like: "Currently selected items are managed by Elastic and cannot be deleted". WDYT?

Reply:
Also, can we disable this button for all hidden or managed SOs?

Reply:
Right now it is possible to get to this slightly awkward state:

<img width="794" alt="Screenshot 2024-02-08 at 12 37 52" src="https://github.com/elastic/kibana/assets/8155004/12cac0ba-02a4-425d-9685-f65a45c2cf0f">

Reply:
You're right. This is a strange state. Although, I guess this is how it already works for hidden content. I agree that hidden and managed content cases should match, but I'm not sure which route to go (i.e. allow the modal to open but not allow deleting, or prevent the modal from opening).

@MichaelMarcialis do you have any input here?